canarytests/agent/src/main/java/emf/canary/ECSRunnable.java

```diff
@@ -13,14 +14,17 @@ public class ECSRunnable implements Runnable {
 
     @Override
     public void run() {
+        final Configuration config = EnvironmentConfigurationProvider.getConfig();
+        config.setEnvironmentOverride(Environments.Agent);
```

Comment:
What does this do?

Reply:
This is a temporary override to skip environment detection. I was using it to test the canary. Will remove.